packages/wallet-backend-update/src/app.ts

```diff
@@ -12,23 +12,49 @@ import type { AuthController } from './auth/controller'
 import type { AuthService } from './auth/service'
 import type { Env } from './config/env'
 import type { UserService } from './user/service'
-import type { UserController } from './user/controller'
 import { Container } from './shared/container'
 import { Model } from 'objection'
 import { withSession } from './middleware/withSession'
 import { errorHandler } from './middleware/errorHandler'
 import type { SessionService } from './session/service'
 import { isAuth } from './middleware/isAuth'
+import { GraphQLClient } from 'graphql-request'
+import { RapydClient } from './rapyd/rapyd-client'
+import { RapydCountriesService } from './rapyd/countries/service'
+import { RapydCountriesController } from './rapyd/countries/controller'
+import { UserController } from './user/controller'
+import { RapydDocumentsController } from './rapyd/documents/controller'
+import { RapydWalletController } from './rapyd/wallet/controller'
+import { RapydWalletService } from './rapyd/wallet/service'
+import { RapyddocumentsService } from './rapyd/documents/service'
+import { AssetController } from './asset/controller'
+import { RafikiClient } from './rafiki/rafiki-client'
+import { AccountService } from './account/service'
+import { AccountController } from './account/controller'
 
 export interface Bindings {
   env: Env
   logger: Logger
   knex: Knex
+  gqlClient: GraphQLClient
+  rapydClient: RapydClient
+  rafikiClient: RafikiClient
+  rapydCountriesService: RapydCountriesService
+  rapydDocumentsService: RapyddocumentsService
+  rapydWalletService: RapydWalletService
```

Comment:
Everything related to Rapyd should live in `rapydService` and everything related to Rafiki should live in `rafiki`/`rafikiService`. I don't think we need to expose the GraphQL Client and can only live in the rafiki service.

Reply:
I would not tie together the rafiki and graphql client, as future proof in case other graphql api might be used, we could use the same gql service.

Everything that makes part of rapyd should be in the rapyd service, indeed.

Reply:
Also I want to add that I didn't move all the rapyd related stuff together is so that I keep the previous routes that Rico had, so there would be no changes to the frontend yet. 
If we move them together, I think this PR also needs to adress the frontend url's used in requests